presto-cli/src/main/java/com/facebook/presto/cli/ClientOptions.java

```diff
@@ -119,6 +119,9 @@
     @Option(name = "--client-request-timeout", title = "client request timeout", description = "Client request timeout (default: 2m)")
     public Duration clientRequestTimeout = new Duration(2, MINUTES);
 
+    @Option(name = "--stop-on-error", title = "stop-on-error", description = "Whether to stop execution immediately if any query fails in batch mode (default: true)")
```

Comment:
without adding `arity = 1` I cannot pass `--stop-on-error false` to the cli, in other words this doesn't work:

```
./presto-cli-0.188-SNAPSHOT-executable.jar --stop-on-error false --schema tpch --catalog sf1 --execute "select 1; select x; select 2;"
Exception in thread "main" io.airlift.airline.ParseArgumentsUnexpectedException: Found unexpected parameters: [false]
	at io.airlift.airline.SingleCommand.validate(SingleCommand.java:98)
	at io.airlift.airline.SingleCommand.parse(SingleCommand.java:65)
	at io.airlift.airline.SingleCommand.parse(SingleCommand.java:56)
	at com.facebook.presto.cli.Presto.main(Presto.java:25)

```